cherrypy/process/servers.py

```diff
@@ -185,7 +185,7 @@ def _get_base(self):
         if not self.httpserver:
             return ''
         host, port = self.bind_addr
-        if getattr(self.httpserver, 'ssl_adapter', None):
+        if getattr(self,'ssl_certificate', None):
```

Comment:
It passed the tests now. However I will try to check the cause for previous failure.